packages/bruno-app/src/utils/exporters/openapi-spec.js

```
@@ -254,69 +279,72 @@ export const exportApiSpec = ({ variables, items, name, environments }) => {
               };
               pathBody['requestBody'] = {
-                $ref: `#/components/requestBodies/${requestBodyId}`
+                $ref: `#/components/requestBodies/${componentId}`
               };
             } catch (error) {
               addWarning(`Failed to parse XML in request body: ${error.message}`, item?.name);
             }
             break;
           case 'multipartForm':
             if (!body?.multipartForm) break;
+            const multipartFormComponentId = getItemComponentId();
```

Biome (2.4.15)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)